kuma/static/styles/components/wiki/indicators.scss

```diff
@@ -86,6 +86,11 @@ modifications for 'system' indicators
     }
 }
 
+.primary {
```

Comment:
We have an existing system for notifications as well as variables defined for colours, there's no need to add new things to the style sheets for this.